source/code/include/playfab/PlayFabCallRequestContainerBase.h

```diff
@@ -38,20 +39,23 @@ namespace PlayFab
         void SetRequestId(const std::string& newRequestId);
         std::string GetRequestBody() const;
         std::shared_ptr<PlayFabApiSettings> GetApiSettings() const;
+        std::shared_ptr<PlayFabAuthenticationContext> GetContext() const;
 
         /// <summary>
         /// This function is meant to handle logic of calling the error callback or success
         /// </summary>
         CallRequestContainerCallback GetCallback() const;
 
         void* GetCustomData() const;
+        virtual bool ValidateSettings() = 0;
 
     protected:
         std::string url;
         std::unordered_map<std::string, std::string> requestHeaders;
         std::string requestBody;
         std::string requestId;
-        std::shared_ptr<PlayFabApiSettings> apiSettings;
+        std::shared_ptr<PlayFabApiSettings> m_settings;
```

Comment:
>m_settings [](start = 44, length = 10)

Is it then possible to make all member variables to be m_* for consistency?

Reply:
That might be reasonable, but it'll also blow up this PR to be "almost every line in these files".
Separate PR?

---
In reply to: [343374534](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343374534) [](ancestors = 343374534)

Reply:
that is going to break API

---
In reply to: [343389603](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343389603) [](ancestors = 343389603,343374534)